projects/valtimo/dossier-management/src/lib/dossier-management-detail-container/dossier-management-detail-container.component.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2015-2020 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Component} from '@angular/core';
+import {ConfigService} from '@valtimo/config';
+
+@Component({
+  selector: 'valtimo-dossier-management-detail-container',
+  templateUrl: './dossier-management-detail-container.component.html',
+  styleUrls: ['./dossier-management-detail-container.component.css']
+})
+export class DossierManagementDetailContainerComponent {
+  public isCase = true;
+  readonly showCaseSearchFields!: boolean;
+
+  constructor(private readonly configService: ConfigService) {
+    this.showCaseSearchFields = configService.config.featureToggles.caseSearchFields;
+  }
+
+  displayBodyComponent(tab: string): void {
+    (this.isCase = tab === 'case')
```

Comment:
The ternary operator (?..:) is not necessary. The assignment should be enough:
`this.isCase = tab === 'case';`

Maybe for readability add some parentheses like this:
`this.isCase = (tab === 'case');`